### ISSUE TITLE
Read multiple blocks command error

### ISSUE THREAD

**Issue by psy2k** [gui-dos/DiaBLE#7]
Sometimes I get an error when scanning in readMultipleBlocks code (blocks returned are 317 while the expected is 344).  Any idea why this is happening?

**Comment by gui-dos** [comment]
There is the CoreNFC method `readMultipleBlock(readConfiguration: NFCISO15693ReadMultipleBlocksConfiguration, completionHandler: @escaping (Data, Error?) -> Void)` which lets you specify the number or retries and the TimeInterval between them.

I never tested it because there is no updated version which returns a `Result<[Data], Error>. It seems it was already available in iOS 11.0 to selected partners as Abbott: let me try to find some samples on how to set correctly the interval between the retries.

**Comment by psy2k** [comment]
Thanks a lot for your prompt response. Is this a documented API?

**Comment by gui-dos** [comment]
Unfortunately the Quick Help is not generated correctly from the headers and there is no mention in the indexed offline or online documentation.

**Comment by psy2k** [comment]
I am trying something like this

```
let command = NFCISO15693ReadMultipleBlocksConfiguration(range:  NSRange(UInt8(blockToRead) ... UInt8(Int(blockToRead) + requested - 1)), chunkSize: 8, maximumRetries: 3, retryInterval: 0.1)
self.connectedTag?.readMultipleBlock(readConfiguration: command, completionHandler: { (blockArray, error)
```

But I get an error that the feature is not supported on this tag. Is the chunk size 8?

**Comment by gui-dos** [comment]
Yep: that cryptic error message was the reason why I gave up using that method after trying all possible combinations…
